client/my-sites/checkout/composite-checkout/components/wp-order-review-line-items.tsx

```diff
@@ -142,7 +142,20 @@ WPOrderReviewLineItems.propTypes = {
 	createUserAndSiteBeforeTransaction: PropTypes.bool,
 };
 
-function canItemBeDeleted( item: ResponseCartProduct ): boolean {
+function canItemBeDeleted(
+	item: ResponseCartProduct,
+	cartProducts: Array< ResponseCartProduct >
+): boolean {
 	const itemTypesThatCannotBeDeleted = [ 'domain_redemption' ];
-	return ! itemTypesThatCannotBeDeleted.includes( item.product_slug );
+	if ( itemTypesThatCannotBeDeleted.includes( item.product_slug ) ) {
+		return false;
+	}
+
+	// The Premium plan cannot be deleted when in combination with the DIFM lite product
+	const isPremiumPlanProduct = item.product_slug === 'value_bundle';
```

Comment:
Should we use `isPremium` ([Link](https://github.com/Automattic/wp-calypso/blob/trunk/packages/calypso-products/src/is-premium.ts#L5)) and `isDIFMProduct` ([Link](https://github.com/Automattic/wp-calypso/blob/trunk/packages/calypso-products/src/is-difm-product.ts#L6)) instead of the string comparison?

Reply:
Ah nice suggestion ! Will get this change in!